### PR TITLE
Added the ID for the NoSQL add-on and a new constructor for the VariantAbstractRPCQuery class

### PR DESCRIPTION
**Pull request by LuigiCasciaro**

### CODE REVIEW

File: src/org/parosproxy/paros/core/scanner/NameValuePair.java [hunk 1]
```diff
@@ -57,6 +58,13 @@
      * @since TODO add version
      */
     public static final int TYPE_MULTIPART_DATA_FILE_CONTENTTYPE = 36;
+    /**
+     * The content-type portion of a application/json file parameter
```
Comment (thc202) [inline src/org/parosproxy/paros/core/scanner/NameValuePair.java]: Might need a tweak, something like `An application/json parameter.`?

**Comment by thc202** [comment]
Thanks!

**Comment by kingthorin** [comment]
Sorry, I should have squashed when merging. I noticed the second commit too late :(

**Comment by thc202** [comment]
No worries :)

**Pull request merged**